build.sbt

```diff
@@ -242,6 +242,20 @@ lazy val integrationTests = (project in file("integration-tests"))
     )
   )
 
+// the chisel standard library
+lazy val standardLibrary = (project in file("stdlib"))
+  .dependsOn(chisel)
+  .settings(commonSettings: _*)
+  .settings(chiselSettings: _*)
+  .settings(usePluginSettings: _*)
+  .settings(
+    Seq(
+      libraryDependencies += defaultVersions("chiseltest") % "test",
+      Compile / scalaSource := baseDirectory.value / "src",
+      Test / scalaSource := baseDirectory.value / "test",
```

Comment:
Am I correct that this is using the Mill directory structure rather than the SBT standard? I don't really have a strong opinion on directory structures other than that they should be consistent within the codebase.

Reply:
sorry about that, we can easily change that back

Reply:
My bad sorry. I’ll change it back.

Reply:
I think I might have started this and not thought enough about consistency. Would be awesome if you could PR a fix, @sequencer

Reply:
I think #2518 fixed this.